corehq/ex-submodules/couchforms/tests/data/devicelogs/devicelog.xml

```diff
@@ -59,7 +59,7 @@
           XPath evaluation: you messed up some types or something
       </msg>
       <session>frame: (COMMAND_ID m2) \ (COMMAND_ID m2-f1)</session>
-      <version>604</version>
+      <app_build>604</app_build>
```

Comment:
If you scroll up a bit, there's another `user_error` entry which I left with `version`, this way tests will encounter both names.